.circleci/config.yml

```diff
@@ -664,15 +678,6 @@ workflows:
               branches:
                 only:
                 - staging
-      - hubploy/build-image:
```

Comment:
was this removed on purpose?

Reply:
I think it was a mistake while editing the config file. Fixed it in the upcoming commit